{
  "id": "12346177",
  "name": "Auth Core 1.4.6",
  "description": "",
  "date": "2020-04-12",
  "version": "1.4.6",
  "module": "Auth Core",
  "lastUpdated": 1633103752308,
  "type": "release",
  "_links": {
    "self": {
      "href": "https://apis.danklco.com/apache-release-info/SLING/12346177"
    },
    "jira": {
      "href": "https://issues.apache.org/jira/browse/SLING/fixforversion/12346177"
    },
    "project": {
      "href": "https://apis.danklco.com/apache-release-info/SLING"
    }
  },
  "releaseNotes": "Release Notes - Sling - Version Auth Core 1.4.6",
  "_embedded": {
    "issues": [
      {
        "issue": "SLING-9345",
        "title": "Update to parent 38 and OSGi DS annotations",
        "type": "Sub-task",
        "_links": {
          "release": {
            "href": "https://apis.danklco.com/apache-release-info/SLING/12346177"
          },
          "jira": {
            "href": "https://issues.apache.org/jira/browse/SLING-9345"
          }
        }
      },
      {
        "issue": "SLING-8711",
        "title": "NPE when auth failed event is sent",
        "type": "Bug",
        "_links": {
          "release": {
            "href": "https://apis.danklco.com/apache-release-info/SLING/12346177"
          },
          "jira": {
            "href": "https://issues.apache.org/jira/browse/SLING-8711"
          }
        }
      }
    ]
  }
}